scenario/elasticuploadapp.go

```diff
@@ -27,15 +27,17 @@ type (
 		MaxRetries int        `json:"retries" displayname:"Number of retries on failed chunk upload" doc-key:"elasticuploadapp.retries"`
 		Mode       UploadMode `json:"mode" displayname:"Upload mode" doc-key:"elasticuploadapp.mode"`
 		Filename   string     `json:"filename" displayname:"Filename" displayelement:"file" doc-key:"elasticuploadapp.filename"`
-		SpaceID    string     `json:"spaceid" displayname:"Space ID" doc-key:"elasticuploadapp.spaceid"`
+		// Deprecated: This property will be removed. DestinationSpace should be used instead, keeping entry here for some time to make sure scripts get validation error
+		SpaceID string `json:"spaceid" displayname:"Space ID" doc-key:"elasticuploadapp.spaceid"`
+		DestinationSpace
 		CanAddToCollection
 	}
 
 	CanAddToCollection struct {
 		Title      session.SyncedTemplate `json:"title" displayname:"Title" doc-key:"elasticuploadapp.title"`
 		Stream     session.SyncedTemplate `json:"stream" displayname:"Stream name" doc-key:"elasticuploadapp.stream"`
 		StreamGUID string                 `json:"streamguid" displayname:"Stream ID" doc-key:"elasticuploadapp.streamguid"`
-		// Deprecated: This property will be removed.ElasticShareApp action should be used instead, keeping entry here for some time to make sure scripts get validation error.
+		// Deprecated: This property will be removed. ElasticShareApp action should be used instead, keeping entry here for some time to make sure scripts get validation error.
 		Groups []string `json:"groups" displayname:"Groups - deprecated" doc-key:"canaddtocollection.groups"`
```

Comment:
should update this reference as share app will disappear. Maybe even time to remove the Groups check?

Reply:
I was thinking about that as well. Might be time.